test/runner.js

```diff
@@ -234,7 +234,7 @@ test('skip test', t => {
 
 	t.throws(() => {
 		runner.skip('should be a todo');
-	}, {message: 'Expected an implementation. Use `test.todo()` for tests without an implementation.'});
+	}, TypeError, 'Expected an implementation. Use `test.todo()` for tests without an implementation.');
```

Comment:
This is incorrect. You're no longer testing the error message. The third argument is just an assert failure message.

You could do:

```js
t.throws(() => {
	runner.todo('todo', () => {});
}, new TypeError('`todo` tests are not allowed to have an implementation. Use `test.skip()` for tests with an implementation.'));
```